docs/conf.py

```diff
@@ -33,7 +34,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "sgkit"
-copyright = "2020, sgkit developers"
+copyright = "{}, sgkit developers".format(datetime.utcnow().year)
```

Comment:
```suggestion
copyright = "2020--{}, sgkit developers".format(datetime.utcnow().year)
```
I think we want to claim the copyright from 2020 until now, this should work